spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2018 Eclipse Microprofile Contributors:
+// Mark Struberg
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+== Configuration
+
+MicroProfile OpenTracing project leverages MicroProfile Config specification to provide a consistent means for
+all supported configuration options. It means that vendor implementations must by also compliant with
+the Config specification.
+
+All configuration keys supported by this projects start with `mp.opentracing`.
+
+=== Configuration items
+
+[cols="1,4"]
+|===
+| Configuration key | Description
+
+| `mp.opentracing.server.skip-pattern` | Configuration property to specify a skip pattern to avoid tracing of selected REST endpoints. See <<server-skip-pattern>>.
```

Comment:
May be drop "Configuration property to" from each entry?

Reply:
changed to `Specifies a skip pattern `...